modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/rel/AbstractIgniteJoin.java

```diff
@@ -90,7 +90,7 @@ protected AbstractIgniteJoin(RelOptCluster cluster, RelTraitSet traitSet, RelNod
         // TODO set NullDirection.LAST for insufficient fields instead of erasing collation.
```

Comment:
TODO without issue. Please create the issue and add to the comment.

Reply:
I've removed this, since it was outdated